.github/workflows/build-main.yml

```diff
@@ -58,6 +58,18 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
+          
+      - name: Cache local Maven repository
+        uses: actions/cache@v2.1.4
+        env:
+          cache-name: cache-mvn
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
```

Comment:
Is trailing dash required or would the following be sufficient as well? It looks a little bit weird compared to other pipeline definition files on GitLab or Azure. :) 

```
${{ runner.os }}-build-${{ env.cache-name }}
${{ runner.os }}-build
${{ runner.os }}
```

Reply:
Just followed the documentation. Probably not needed.
https://github.com/actions/cache/blob/main/examples.md#java---maven

Reply:
okay, then it looks good to me :) Nice work! Thank you very much!